cdk_framework/EKS/lib/utils/validate-interface-schema.ts

```diff
@@ -4,7 +4,7 @@ import { ec2ClusterInterface } from "../interfaces/ec2cluster-interface"
 const validateSchema = require('yaml-schema-validator')
 
 const supportedLaunchTypes = new Set(['fargate', 'ec2'])
-const supportedVersions = new Set(['1.18', '1.19', '1.20', '1.21']);
+const supportedVersions = new Set(['1.18', '1.19', '1.20', '1.21','1.22','1.23']);
```

Comment:
```suggestion
const supportedVersions = new Set(['1.21','1.22','1.23']);
```

Aren't all versions prior to 1.21 deprecated and unable to be used for new clusters?

Reply:
Good catch, thanks.